src/ApiCalls/removeLikeVideo.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+export const removeLikeVideo = async (id, handleaddtoast, setUser_Data) => {
+  const loginToken = localStorage.getItem("token");
+  try {
+    const response = await axios({
```

Comment:
you can move the api calls into a diff folder called services.

so videoService.js could contain all the api calls for the videos.

it is good practice move all the api calls into a services folder and keep the logic and api call separate

Reply:
ApiCalls Folder itself is for separate API calls.. Have used this removeLikeVideo in video card component.